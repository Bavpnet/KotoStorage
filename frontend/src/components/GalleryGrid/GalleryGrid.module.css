.galleryGrid {
  display: grid;
  justify-content: center;
  align-items: center;
  width: 100%;
  grid-gap: 32px;
  grid-template-columns: repeat(auto-fill, 360px);
}

.galleryGrid > img {
  width: 360px;
  height: 360px;
  border-radius: 10px;
  object-fit: cover;
}
