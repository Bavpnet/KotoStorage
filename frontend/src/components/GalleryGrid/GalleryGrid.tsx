import { useCats } from '../../hooks/useCats'
import styles from './GalleryGrid.module.css'

export const GalleryGrid: React.FC = () => {
  const cats = useCats()
  return (
    <div className={styles.galleryGrid}>
      {cats.map(cat => (
        <img key={cat.id} src={cat.url} alt={cat.id} />
      ))}
    </div>
  )
}
