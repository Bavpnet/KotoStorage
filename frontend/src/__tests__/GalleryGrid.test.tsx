import renderer from 'react-test-renderer'
import { GalleryGrid } from '../components/GalleryGrid'

describe('GalleryGrid', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<GalleryGrid />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
