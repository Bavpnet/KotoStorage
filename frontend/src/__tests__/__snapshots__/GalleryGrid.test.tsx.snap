// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GalleryGrid renders correctly 1`] = `
<div
  className="notFound"
>
  <h3>
    Cats not found
  </h3>
</div>
`;
